library/swift/test/EngineBuilderTests.swift

```diff
@@ -199,6 +199,20 @@ final class EngineBuilderTests: XCTestCase {
     self.waitForExpectations(timeout: 0.01)
   }
 
+  func testAddingStringAccessorToConfigurationWhenRunningEnvoy() throws {
+    let expectation = self.expectation(description: "Run called with expected data")
+    MockEnvoyEngine.onRunWithConfig = { config, _ in
+      XCTAssertEqual(1, config.stringAccessors.count)
+      expectation.fulfill()
+    }
+
+    _ = try EngineBuilder()
+      .addEngineType(MockEnvoyEngine.self)
+      .addStringAccessor(name: "name", accessor: { return "hello"})
```

Comment:
```suggestion
      .addStringAccessor(name: "name", accessor: { "hello" })
```
This is equivalent if you want